input/templates/my-account-address-book.hbs

```diff
@@ -34,61 +34,34 @@
                       <div class="box-addresses-title">
                         {{i18n "my-account:addressBook.defaultAddresses.title"}}
                       </div>
+                      {{#if content.defaultShippingAddress}}
                       <div class="address-book-dark-box">
-                        <div class="address-book-dark-box-title">{{i18n "my-account:addressBook.normalAddress.title"}}</div>
-                        <div>{{content.shippingAddress.name}}</div>
-                        <div>{{content.shippingAddress.address}}</div>
-                        <div>{{content.shippingAddress.region}}</div>
-                        <div>{{content.shippingAddress.postalCode}}</div>
-                        <div>{{content.shippingAddress.country}}</div>
-                        <div class="address-box-edit-btn">
-                          <a href="{{@root.meta._links.editMyAddressBook.href}}"><img src="{{meta.assetsPath}}/img/edit-1.png" alt="edit icon">{{i18n "my-account:buttons.edit"}}</a>
-                        </div>
+                        <div class="address-book-dark-box-title">{{i18n "my-account:addressBook.defaultShippingAddress.title"}}</div>
```

Comment:
Not sure if it was a mistake, but this is the "shipping address", not a "normal address".